job_service/model/request.py

```diff
@@ -103,9 +103,7 @@ class MaintenanceStatusRequest(CamelModel, extra=Extra.forbid):
 
     @root_validator(skip_on_failure=True)
     def check_values(cls, values):  # pylint: disable=no-self-argument
-        msg = values["msg"]
-        pause = values["pause"]
-        if not (msg and pause):
+        if not ("msg" in values and "pause" in values):
```

Comment:
Since these are not typed as optional the model will throw a validation error if they are not present 👍🏻 Pydantic will handle it if either of these fields are missing or None, so this root validator is not necessary

Reply:
Thanks, I see your point, I fix